### ISSUE TITLE
Update iptables to >= 1.8.8

### ISSUE THREAD

**Issue by procinger** [flatcar/Flatcar#954]
## Description

The version of `iptables 1.8.7` delivered in `Flatcar` does not seem to work correctly. There is an option `-C` to check if a rule exists in a chain/table, but it does not always return the correct result. Instead of saying that the rule exists, it sometimes returns the message that said rule does not exist. 

This has the consequence that scripts erroneously create the same rules over and over again and the systems gets a high iowait at some point.

In the included toolbox there is iptables in version 1.8.8 which always returns the correct result. To verify this, I built a local Qemu image with Flatcar 3374.2.3 and iptables version 1.8.8, after that I could not reproduce the behavior anymore.

## Impact

The automatic creation of rules is impossible, because sometimes  iptables does not find the said rule during an exists check. Thus rules would be created multiple times, which leads to a high iowait, because the ksoftirq processes takes longer to run

## Environment and steps to reproduce

1. **Set-up**: Install Flatcar 3374.2.3 on a machine
2. **Task**:  create iptables rule `iptables -I INPUT -p tcp -d 10.20.30.40 --dport 53 -j ACCEPT`
4. **Action(s)**: 
  a. check if rule exists `iptables -v -C INPUT -t filter -p tcp -d 10.20.30.40 --dport 53 -j ACCEPT`
5. **Error**: the rule is sometimes not found `iptables: Bad rule (does a matching rule exist in that chain?).`

## Expected behavior
The rule should always be found.

```
flatcar# iptables -v -C INPUT -t filter -p tcp -d 10.20.30.40 --dport 53 -j ACCEPT
ACCEPT  tcp opt -- in * out *  0.0.0.0/0  -> 10.20.30.40   tcp dpt:53
flatcar# iptables -v -C INPUT -t filter -p tcp -d 10.20.30.40 --dport 53 -j ACCEPT
iptables: Bad rule (does a matching rule exist in that chain?).
flatcar# iptables -v -C INPUT -t filter -p tcp -d 10.20.30.40 --dport 53 -j ACCEPT
ACCEPT  tcp opt -- in * out *  0.0.0.0/0  -> 10.20.30.40   tcp dpt:53
```
## Additional information

I could not see a pattern of when the error is thrown. Sometimes it happens at the first iptables check, sometimes at the thousandth. Therefore maybe use a while true loop when checking.

**Comment by dongsupark** [comment]
Thanks for the detailed report.
We will soon try to update iptables to 1.8.8 in Alpha.

However, it is for me not trivial to reproduce the "Bad rule" issue on my Qemu VM of Stable 3374.2.3. Have tried with various options for hours, but no luck so far.
It would be great if we could have a simple reproducer script.
That could help us to decide if the new iptables version is something we should also have in Stable/Beta channels.

**Comment by dongsupark** [comment]
PR https://github.com/flatcar/coreos-overlay/pull/2433 was merged, so iptables 1.8.8 will be available in the next Alpha.

**Comment by procinger** [comment]
hi @dongsupark , thanks for taking care of the problem. I will see if I can find a better / more effective iptables test

**Comment by dongsupark** [comment]
Recently Stable 3510.2.0 and 3510.2.1 were released with iptables 1.8.8.
This issue should be resolved in Stable, Beta, and Alpha.
Thanks.
